workers/unity/Packages/com.improbable.gdk.buildsystem/Util/BuildSupportChecker.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+public static class BuildSupportChecker
+{
+    private static readonly Dictionary<BuildTarget, string> BuildPlatformSupportModuleDirectoryNames;
+
+    static BuildSupportChecker()
+    {
+        BuildPlatformSupportModuleDirectoryNames = new Dictionary<BuildTarget, string>();
+
+        const string linuxStandaloneSupport = "LinuxStandaloneSupport";
+
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux64] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinuxUniversal] = linuxStandaloneSupport;
+
+#if !UNITY_EDITOR_OSX
```

Comment:
Again, why conditional compilation? It looks like this should work just by checking the platform.

Reply:
Removing conditional compilation and replacing with platform checks